vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/InputProvisioner.java

```diff
@@ -24,6 +24,10 @@
   /**
    * Begin provisioning out a new input that was registered in Vidarr
    *
+   * <p>This method should not do any externally-visible work. Anything it needs should be done
+   * should be done in a {@link WorkMonitor#scheduleTask(Runnable)} callback so that Vidarr can
+   * execute it once the database is in a healthy state.
```

Comment:
* `<p>` not closed
* "should be done should be done"

Same for below, and in other classes this was copied to